eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Generator/Update.hs

```diff
@@ -100,7 +100,7 @@ genPParams ::
   (EraPParams era, ProtVerAtMost era 4, ProtVerAtMost era 6) =>
   Constants ->
   Gen (PParams era)
-genPParams c@Constants {maxMinFeeA, maxMinFeeB, minMajorPV} = do
+genPParams c@Constants {maxMinFeeA, maxMinFeeB, minMajorPV, maxMajorPV} = do
```

Comment:
I'm surprised we can get away with updating the max major protocol here without a ton of headache...

Reply:
It's the new one `maxMajorPV` in Constants that I introduced, not the existing one in `Globals`. I haven't managed to fully understand the impact of changing that one, so I decided it's better to control how we generate the data than to tune it from that flag..